package.json

```diff
@@ -32,24 +32,26 @@
   "main": "./dist/index.js",
   "typings": "./dist/index.d.ts",
   "dependencies": {
-    "@angular/common": "2.0.0",
-    "@angular/compiler": "2.0.0",
-    "@angular/core": "2.0.0",
-    "@angular/http": "2.0.0",
-    "@angular/platform-browser": "2.0.0",
-    "@types/lodash": "^4.14.37",
-    "@types/reflect-metadata": "0.0.4",
-    "core-js": "^2.4.1",
-    "lodash": "^4.16.4",
-    "reflect-metadata": "0.1.3",
-    "rxjs": "5.0.0-beta.12",
-    "zone.js": "0.6.23"
+    "lodash": "^4.16.4"
+  },
+  "peerDependencies": {
+    "@angular/http": "^2.0.0",
+    "@angular/core": "^2.0.0",
+    "reflect-metadata": "^0.1.3",
+    "rxjs": "5.0.0-beta.12"
   },
   "devDependencies": {
+    "@angular/core": "^2.0.0",
+    "@angular/common": "^2.0.0",
+    "@angular/compiler": "^2.0.0",
+    "@angular/http": "^2.0.0",
+    "@angular/platform-browser": "^2.0.0",
+    "@angular/platform-browser-dynamic": "^2.0.0",
+    "@types/lodash": "^4.14.37",
```

Comment:
If we put the angular packages to `dependencies`, npm would possibly install these packages twice and then the packages used in `angular2-jsonapi` can differ from the ones used in consuming packages. 
I looked at how other angular2 libraries solve this issue, for example here: https://github.com/flauc/angular2-notifications/blob/master/package.json
The @types/lodash package is only required in dev, because it contains no code.